package.json

```diff
@@ -113,8 +113,7 @@
     "cacheDir": ".changelog",
     "labels": {
       "PR: Spec Compliance :eyeglasses:": ":eyeglasses: Spec Compliance",
-      "PR: Breaking Change :boom:": ":boom: Breaking Change",
-      "PR: Deprecation: :loudspeaker:": ":loudspeaker: Deprecation",
+      "PR: Deprecation :loudspeaker:": ":loudspeaker: Deprecation",
```

Comment:
Previously the `Deprecation` PR was never included in the change log because of the extra colon. This is now fixed.